remove --socks5ProxyXmrAddress from external tor docs

woodser opened the pull request.

PromptPunksFauxCough commented:
Removing it from docs doesn't do much as, it's still available and easily discovered with `--help` option.

IMO, a footnote mentioning current inconsistencies with functionality referencing the open issue might make more sense.

PromptPunksFauxCough commented:
Please see [here](https://github.com/haveno-dex/haveno/issues/1649#issuecomment-2711314275) for more detail.

boldsuck commented:
> IMO, a footnote mentioning current inconsistencies with functionality referencing the open issue might make more sense.

In Haveno Docs I put optional Options in parentheses and added a note:
https://boldsuck.github.io/haveno-docs/haveno-ui/external-tor-usage/#starting-haveno-using-externally-available-tor

I also changed the link from the Tor Project website directly to Install instructions and added some notes.

woodser commented:
> In Haveno Docs I put optional Options in parentheses and added a note: https://boldsuck.github.io/haveno-docs/haveno-ui/external-tor-usage/#starting-haveno-using-externally-available-tor
> 
> I also changed the link from the Tor Project website directly to Install instructions and added some notes.

I'm getting a weird rendering of the link, but do you mind updating the main copy of this doc directly? Otherwise they're going to get out of sync: https://github.com/haveno-dex/haveno/blob/master/docs/external-tor-usage.md

boldsuck commented:
> I'm getting a weird rendering of the link,

;-)
Yes, sorry. After writing this, I committed your suggestion to 'change section titled "Tor configuration"'

https://boldsuck.github.io/haveno-docs/tor-config/external-tor-usage/#directbindtor-static-onion-assignment-via-hiddenserviceaddress

> but do you mind updating the main copy of this doc directly? Otherwise they're going to get out of sync

Yes, of course.
For mkdocs, I have to significantly change the Markdown formatting.
I commit the text changes to https://github.com/haveno-dex/haveno/blob/master/docs/external-tor-usage.md

woodser commented:
No longer necessary with fix: https://github.com/haveno-dex/haveno/pull/1658
